cinema/urls.py

```diff
@@ -1 +1,20 @@
-# write urls here
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from .views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+)
+
+router = DefaultRouter()
+router.register(r"genres", GenreViewSet)
+router.register(r"actors", ActorViewSet)
+router.register(r"cinema_halls", CinemaHallViewSet)
+router.register(r"movies", MovieViewSet)
+router.register(r"movie_sessions", MovieSessionViewSet)
```

Comment:
`r` przed stringiem w tym kontekście nic nie zmieniają, wiec moża się ich pozbyć
```suggestion
router.register("genres", GenreViewSet)
router.register("actors", ActorViewSet)
router.register("cinema_halls", CinemaHallViewSet)
router.register("movies", MovieViewSet)
router.register("movie_sessions", MovieSessionViewSet)
```